stock_reception_dedicate_picking_type/views/stock_warehouse.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright (C) 2022 Akretion (<http://www.akretion.com>).
+     @author Kévin Roche <kevin.roche@akretion.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
```

Comment:
As it this file is useless but I suppose you wanted to add  stock_in_type_id and quality_type_id fields on the warehouse form view ?